lib/bundle_update_interactive/cli.rb

```diff
@@ -1,40 +1,33 @@
 # frozen_string_literal: true
 
-require "thor"
+require "bundler"
 
 module BundleUpdateInteractive
-  class CLI < Thor
+  class CLI
     autoload :MultiSelect, "bundle_update_interactive/cli/multi_select"
+    autoload :Options, "bundle_update_interactive/cli/options"
     autoload :Row, "bundle_update_interactive/cli/row"
     autoload :Table, "bundle_update_interactive/cli/table"
-    autoload :ThorExt, "bundle_update_interactive/cli/thor_ext"
 
-    extend ThorExt::Start
+    def run(argv: ARGV) # rubocop:disable Metrics/AbcSize
+      Options.parse(argv)
```

Comment:
❔ Down the line when there are CLI flags that can influence the report, do you see this returning some sort of data structure that gets passed to `generate_report`?

Reply:
@dylnclrk yes, I see it returning an instance of `CLI::Options`. Right now that's an empty class, but it could expand if/when we add flags in the future. For example, if we were to add a `--quiet` flag, then we could add a corresponding attribute to the class:

```ruby
class CLI::Options
  attr_writer :quiet

  def initialize
    # Set defaults
    @quiet = false
  end

  def quiet?
    @quiet
  end
end
```

I'll push a commit to this PR to make `CLI::Options.parse` return an instance of the class, for future use.